src/styles/index.scss

```diff
@@ -1,3 +1,258 @@
-body {
+@import 'variables';
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: $fw-regular;
+  line-height: 18px;
+  color: $main-text-color;
+}
+
+.container {
+  max-width: 1300px;
+  margin: 0 auto;
+  display: grid;
+
+  box-sizing: border-box;
+}
+
+.header {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__box {
+    max-width: 1200px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    padding: 0 50px;
+    height: 60px;
+  }
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
+  &__menu {
+    display: flex;
+    gap: 20px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__link {
+    display: inline-block;
+    align-items: center;
+    height: 60px;
+    text-decoration: none;
+    color: $main-text-color;
```

Comment:
Ensure that the variable `$main-text-color` is defined in the 'variables' file. If not, it may cause a compilation error.